packages/amplify-provider-awscloudformation/src/aws-utils/aws.js

```diff
@@ -1,5 +1,15 @@
-process.env.AWS_SDK_LOAD_CONFIG = true;
-const aws = require('aws-sdk');
+let aws;
+
+try {
+  delete require.cache[require.resolve('aws-sdk')];
```

Comment:
why are we deleting this even before requiring this?

Reply:
To ensure consistent behavior, if aws-sdk is previously required, it'll be in the cache, and the new require won't really kick off, then the process.env.AWS_SDK_LOAD_CONFIG = true; setting is nullified.

Reply:
How could someone require `aws-sdk`? Don't we use this codepath to be executed all the time?

Reply:
this is a safeguard, because we have a plugin platform, other plugins can require the aws-sdk before the amplify-provider-awscloudformation plugin is loaded.

Reply:
If aws-sdk is loaded beforehand no harm no foul. I don't like us removing stuff from cache if its not necessary

Reply:
Then it's inconsistent behavior, if the the project is configured to use 'general' config level, the default profile won't be read, aws access will be denied.